docs/source/tensor_view.rst

```diff
@@ -0,0 +1,74 @@
+.. currentmodule:: torch
+
+.. _tensor-view-doc:
+
+Tensor Views
+=================
+
+`Tensor` and `Storage` are two core concepts in PyTorch. `Tensor` is the main user facing data structure,
+while `Storage` is mostly internal. Users usually don't have to deal with `Storage` directly.
+
+`Tensor` contains data, as well as its metadata such as size, dtype and device. But `Tensor` doesn't
```

Comment:
nit: Tensor only adds `size` and `stride`. Storage has a fixed `dtype` and `device` already.

Reply:
@albanD Did we document this method somewhere? ;) 
I can write text but curious if I'm missing it